protocol-definitions/Metrics.yaml

```diff
@@ -0,0 +1,40 @@
+id: 39
+name: Metrics
+methods:
+  - id: 1
+    name: readMetrics
+    since: 2.0
+    doc: |
+      Reads the recorded metrics starting with the smallest sequence number
+      greater or equals to the sequence number set in fromSequence.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: name
+      params:
+        - name: uuid
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            The UUID of the member that is supposed to read the metrics from.
+        - name: fromSequence
+          type: long
+          nullable: false
+          since: 2.0
+          doc: |
+            The sequence the recorded metrics should be read starting with.
+    response:
+      params:
+        - name: elements
+          type: Map_Long_byteArray
```

Comment:
Should the metrics be extended or should we use serialized byte array as here? byte array is always dangerous as it is very easy to break compatibility.

Reply:
The byte array is the compressed data of all metrics. So it's by design byte[]